package.json

```diff
@@ -30,9 +30,10 @@
     "build:types": "tsc -b",
     "clean": "run-s clean:*",
     "clean:distfiles": "npx rimraf \"packages/*/build\"",
+    "clean:monorepo": "npx rimraf node_modules package-lock.json",
```

Comment:
this will break `npm run clean` because it will delete the node_modules before running `clean:packages`.

the order here matters; we either need to make sure it stays the way it is, or just spell out which clean scripts to run (i.e., don't use `run-s clean:*`)

Reply:
ah, I had just moved this line, but I've fixed this anyway